hcctl/src/main.rs

```diff
@@ -0,0 +1,60 @@
+#[macro_use]
+extern crate prettytable;
+
+use anyhow::{anyhow, Result};
+use healthchecks::manage;
+use prettytable::{format, Table};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Opts {
+    /// API token to use when interacting with the healthchecks API
+    #[structopt(
+        long = "api-token",
+        global = true,
+        env = "HEALTHCHECKS_API_TOKEN",
+        // This is annoying but you can't set a global option without a default
+        default_value = ""
+    )]
+    api_token: String,
+
+    #[structopt(subcommand)]
+    command: Command,
+}
+
+#[derive(Debug, StructOpt)]
+enum Command {
+    /// List the checks in your account
+    List,
+}
+
+fn main() -> Result<()> {
```

Comment:
I prefer using `anyhow::Result` directly as opposed to importing since there's an std `Result` type which can be confusing at first glance.